lib/widgets/message_list.dart

```diff
@@ -24,10 +24,28 @@ class MessageListPage extends StatelessWidget {
 
   final Narrow narrow;
 
+  @override
+  State<MessageListPage> createState() => MessageListPageState();
+}
+
+class MessageListPageState extends State<MessageListPage> {
+  GlobalKey<ComposeBoxController> get composeBoxKey => _composeBoxKey;
```

Comment:
nit: This getter isn't needed; the only caller is `composeBoxControllerOf`, inside this file.